k8s/contiv-vpp.yaml

```diff
@@ -19,48 +21,20 @@ metadata:
   name: contiv-agent-cfg
   namespace: kube-system
 data:
-  contiv.yaml: |
-    TCPstackDisabled: True
-    UseTAPInterfaces: True
+  contiv.yaml: |-
+    TCPstackDisabled: true
+    UseTAPInterfaces: true
     TAPInterfaceVersion: 1
     IPAMConfig:
-      PodSubnetCIDR: "10.1.0.0/16"
+      PodSubnetCIDR: 10.1.0.0/16
       PodNetworkPrefixLen: 24
-      VPPHostSubnetCIDR: "172.30.0.0/16"
+      VPPHostSubnetCIDR: 172.30.0.0/16
       VPPHostNetworkPrefixLen: 24
-      NodeInterconnectCIDR: "192.168.16.0/24"
-      VxlanCIDR: "192.168.30.0/24"
-#      ServiceCIDR: "10.96.0.0/12"
-#      NodeInterconnectDHCP: True
-### example of node configuration for VPP interfaces
-#    NodeConfig:
-#    - NodeName: "vm1"
-#      MainVPPInterface:
-#        InterfaceName: "GigabitEthernet0/4/0"
-#        IP: 192.168.16.101/24
-#      Gateway: 192.168.1.1
-#      OtherVPPInterfaces:
-#      - InterfaceName: "GigabitEthernet0/5/0"
-#        IP: "1.2.3.4/24"
-#      - InterfaceName: "GigabitEthernet0/6/0"
-#        IP: "2.3.4.5/24"
-#    - NodeName: "vm2"
-#      MainVPPInterface:
-#        InterfaceName: "GigabitEthernet0/9/0"
-#        UseDHCP: True
-#      Gateway: 192.168.1.1
-#      OtherVPPInterfaces:
-#      - InterfaceName: "GigabitEthernet0/7/0"
-#        IP: "3.4.5.6/24"
-#      - InterfaceName: "GigabitEthernet0/7/0"
-#        IP: "5.6.7.8/24"
+      VxlanCIDR: 192.168.30.0/24
+      NodeInterconnectDHCP: True
 
```

Comment:
Default for open source should be static IP address assignment

Reply:
Updated.